WordPress/src/main/java/org/wordpress/android/ui/accounts/SignInFragment.java

```diff
@@ -640,7 +564,11 @@ public void onResponse(JSONObject jsonObject) {
                         finishCurrentActivity(userBlogList);
                         String displayName = JSONUtils.getStringDecoded(jsonObject, "display_name");
                         Uri profilePicture = Uri.parse(JSONUtils.getString(jsonObject, "avatar_URL"));
-                        saveCredentialsInSmartLock(mUsername, mPassword, displayName, profilePicture);
+                        SmartLockHelper smartLockHelper = getSmartLockHelper();
+                        if (smartLockHelper != null) {
```

Comment:
If the user uses Magic Links to sign in, `mUsername` and `mPassword` are null, causing a crash.